tests/test-infra/clean_up.sh

```diff
@@ -20,7 +20,7 @@ echo "Trying to delete namespace..."
 kubectl delete namespace $1 --timeout=10m
 
 for pod in `kubectl get pods -n $1 -o name`; do
-	kubectl delete --force -n $1 $pod
+  kubectl delete --force -n $1 $pod
```

Comment:
nitpick: can we have the same amount of space in line 14 as well?

Reply:
done.